asistente_ladm_col/gui/qgis_model_baker/dlg_export_data.py

```diff
@@ -312,9 +314,17 @@ def accepted(self):
                     self.show_message(QCoreApplication.translate("DialogExportData", "An error occurred when exporting the data. For more information see the log..."), Qgis.Warning)
                     return
             except JavaNotFoundError:
-                self.txtStdout.setTextColor(QColor('#000000'))
-                self.txtStdout.clear()
-                self.txtStdout.setText(QCoreApplication.translate("DialogExportData", """Java could not be found. Please <a href="https://java.com/en/download/">install Java</a> and or <a href="#configure">configure a custom java path</a>. We also support the JAVA_HOME environment variable in case you prefer this."""))
+                # Set JAVA PATH
+                get_java_path_dlg = DialogGetJavaPath()
+                get_java_path_dlg.setModal(True)
+                if get_java_path_dlg.exec_():
+                    configuration = self.update_configuration()
+
+                if not get_java_path_from_qgis_model_baker():
+                    self.txtStdout.setTextColor(QColor('#000000'))
+                    self.txtStdout.clear()
+                    self.txtStdout.setText(QCoreApplication.translate("DialogExportData",
+                                                                      """Java could not be found. Please <a href="https://java.com/en/download/">install Java</a> and or <a href="#configure">configure a custom java path</a>. We also support the JAVA_HOME environment variable in case you prefer this."""))
```

Comment:
We could better use the message bar to inform this since the Log panel could be hidden.